data/javascript/currency-format.json

```diff
@@ -2355,7 +2355,7 @@
     "symbol": "UGX",
     "decimal": ".",
     "group": ",",
-    "precision": 0,
+    "precision": 2,
```

Comment:
@paololim Does this look correct? Not sure if there should be a change in javascript.

Reply:
this looks correct, lets just make sure this is the right change we want